### PR TITLE
add bitwise_or to jax

### PR DESCRIPTION
**Pull request by PatricYan**
- [ ] #3110 
- add bitwise_or to jax

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py [hunk 1]
```diff
@@ -717,3 +717,40 @@ def test_jax_lax_bitwise_and(
...
+
+# bitwise_or
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=ivy_jax.valid_int_dtypes,
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+    as_variable=helpers.list_of_length(x=st.booleans(), length=2),
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:730]: please remove this line as it is handled by the `@handle_cmd_line_args` decorator

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py [hunk 2]
```diff
@@ -717,3 +717,40 @@ def test_jax_lax_bitwise_and(
...
+    num_positional_args=helpers.num_positional_args(
+        fn_name="ivy.functional.frontends.jax.lax.bitwise_or"
+    ),
+    native_array=helpers.list_of_length(x=st.booleans(), length=2),
```
Comment (juliagsy) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:734]: please remove this line as it is handled by the `@handle_cmd_line_args` decorator

**Review by juliagsy — CHANGES_REQUESTED** [review]
Hello! Great work so far! Just some small changes and you're good to go! Please request my review once you're done with the changes! Thanks!

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py (hunk 2 above)
Comment (PatricYan) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py]: done

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py (hunk 1 above)
Comment (PatricYan) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py]: done

**Comment by PatricYan** [comment]
> Hello! Great work so far! Just some small changes and you're good to go! Please request my review once you're done with the changes! Thanks!

Hi, I have removed the codes.

**Comment by juliagsy** [comment]
Thanks for contributing! Will merge soon!

**Pull request merged**